app/main/views/services.py

```diff
@@ -361,7 +361,7 @@ def copy_draft_service(framework_slug, lot_slug, service_id):
                             service_id=draft_copy['id'],
                             section_id=section_id_to_edit,
                             question_slug=question_slug_to_edit,
-                            return_to_summary=1
+                            force_continue_button=1
```

Comment:
The existing behaviour is exactly as was intended when the "copy draft service" flow was designed and built.

The intention here - when copying a service - is that the supplier hits "copy" and is taken to the page where they have to edit the name of the copied service.  Once the service name has been edited they then "return to summary" to see the overview of the copied service, which they can then edit sections of individually as they choose.

Reply:
Yeah, it feels a bit weird to me though. Coming from the page where you hit copy, and then seeing 'Save and return to service summary'. You've never been to the service summary for that new service, so you can't return there. 'Return' is back to the page you were on, the service summary list. If you think the original behaviour is ok then I'm happy to just do the button content here, have included a couple of gifs to show the existing and potential new button though:

#### Existing:

![save_and_return_to_service_summary](https://user-images.githubusercontent.com/13836290/37588187-61466b06-2b59-11e8-899d-8e685f6141d7.gif)

#### Potential:

![save_and_continue](https://user-images.githubusercontent.com/13836290/37588169-5a2613ee-2b59-11e8-8606-204cbe8ed0c0.gif)

Reply:
Agreed that "Save and continue" better the first time after a copy.